homeassistant/components/google_generative_ai_conversation/conversation.py

```diff
@@ -240,6 +240,7 @@ async def async_process(
                 user_prompt=user_input.text,
                 language=user_input.language,
                 assistant=conversation.DOMAIN,
+                device_id=user_input.device_id,
```

Comment:
almost makes me wonder if we shouldn't just pass `user_input` ?

Reply:
I guess that would tie it to the conversation integration and we don't want that.